api/specs.proto

```diff
@@ -334,13 +332,55 @@ message NetworkSpec {
 	// the services attached to this network. If this flag is not
 	// enabled(default case) no manual attachment to this network
 	// can happen.
-	bool attachable = 6;
+	bool attachable = 5;
 
 	// Ingress indicates this network will provide the routing-mesh.
 	// In older versions, the network providing the routing mesh was
 	// swarm internally created only and it was identified by the name
 	// "ingress" and the label "com.docker.swarm.internal": "true".
-	bool ingress = 7;
+	bool ingress = 6;
+}
+
+// NetworkSpec specifies user defined network parameters.  All fields
+// with a cnm_compat_ (CNMCompat in go) prefix are deprecated and the
+// contents of the `backend` oneof will be preferred if present.
```

Comment:
Done. You were correct that it didn't do anything other than change `fileDescriptorObjects`.